### DISCUSSION TITLE
LAN and PoE Port on satellite

### DISCUSSION THREAD

**Discussion by Marfinho** [FutureProofHomes/Satellite1-Hardware#3] [Q&A] [answered]
Hey guys,
Exciting announcement with the satellite. Me and some smart home nerds from germany was discussing the reasons for why to rely only on WiFi? Especially with PoE it would be so cool and easy to integrate into homes. 

Rech out if you Need some help for the mounting stuff, I’d Like to contribute :)

cheers,
Sven

**Comment by ghdoca** [comment]
This is exactly what I'm missing. The PoE wire is already in the ceiling, so changing it to USB would be a no go. Otherwise great project!

**Comment by zach6439** [comment]
+1 to PoE, It may be a bit cost prohibitive to have two options, one PoE, one without, but I would probably spring for the PoE version whenever possible.

**Comment by that1guy** [comment]
Hey gents, we will have PoE on the Sat1 enclosure which wraps around the dev kit boards.  Also that PoE implementation will give us networking since we'll be using [ESPHome's Ethernet Component.](https://esphome.io/components/ethernet.html).

Sorry for delayed response, btw.  I've been travelling and our support SLAs haven't been set up quite yet while the team is heads down building. :)

**Comment by that1guy** [comment, marked as answer]
Please read about the upcoming Satellite1 SHOE board we're working on here: https://futureproofhomes.net/blogs/news/q1-2025-status-update

This accessory board will attached to the Satellite1 Dev Kit and expose PoE as well as helping move all the sensors down into the speaker chamber.
